src/test/java/org/candlepin/insights/controller/InventoryControllerTest.java

```diff
@@ -120,6 +120,33 @@ void testHostSkippedWhenExceptionHappens() {
         verify(inventoryService, times(1)).flushHostUpdates();
     }
 
+    @Test
+    void testSkipManifestConsumers() {
+        UUID uuid1 = UUID.randomUUID();
+        UUID uuid2 = UUID.randomUUID();
+        Consumer consumer1 = new Consumer();
+        Consumer consumer2 = new Consumer();
+        consumer1.setUuid(uuid1.toString());
+        consumer1.setAccountNumber("account");
+        consumer1.setOrgId("456");
+        consumer1.setType("system");
+        consumer2.setUuid(uuid2.toString());
+        consumer2.setAccountNumber("account");
+        consumer2.setOrgId("456");
+        consumer2.setType("candlepin");
+        when(pinheadService.getOrganizationConsumers("123")).thenReturn(
+            Arrays.asList(consumer1, consumer2));
+        controller.updateInventoryForOrg("123");
+
+        ConduitFacts expected = new ConduitFacts();
+        expected.setOrgId("456");
+        expected.setAccountNumber("account");
+        expected.setSubscriptionManagerId(uuid1.toString());
+        verify(inventoryService).scheduleHostUpdate(Mockito.eq(expected));
+        verify(inventoryService, times(1)).flushHostUpdates();
+        verifyNoMoreInteractions(inventoryService);
```

Comment:
Just curious, why do we need to `verifyNoMoreInteractions`?

Reply:
If you run the test without the fix, you can observe that `verifyNoMoreInteractions` fails when `scheduleHostUpdate` is called with the manifest consumer.